pkg/kubelet/network/cni/cni.go

```diff
@@ -53,14 +59,18 @@ type cniNetwork struct {
 
 func probeNetworkPluginsWithVendorCNIDirPrefix(pluginDir, vendorCNIDirPrefix string) []network.NetworkPlugin {
 	configList := make([]network.NetworkPlugin, 0)
-	network, err := getDefaultCNINetwork(pluginDir, vendorCNIDirPrefix)
-	if err != nil {
-		return configList
-	}
-	return append(configList, &cniNetworkPlugin{
-		defaultNetwork: network,
+	plugin := &cniNetworkPlugin{
+		defaultNetwork: nil,
```

Comment:
No, I think this is correct. Creating a pod on the pod network should fail until cni is configured. However pods on the hostnetwork shouldn't be affected by this static check.